testing/bsg_dataflow/Makefile

```diff
@@ -0,0 +1,14 @@
+# High Level Makefile
+# 
+# This makefile runs all other makefiles within one level of
+# scope of itself
+#
+# Brenden Page 11/19/2022
+
+TEST_DIRS = $(wildcard ./*/)
+
+all:
+	$(foreach x,$(TEST_DIRS), cd $(x) && make && cd ../;)
```

Comment:
nit; 
$(MAKE) -C $(x)
will preserve -j and other options when passing into the subshell (and prevent the need for a cd)